docker-compose.yml

```diff
@@ -11,8 +11,10 @@ services:
             - python
             - python-sync
             - redis
+            - cherami
         environment:
-            - WAIT_FOR=go,node,python,python-sync
+            - WAIT_FOR=go,node,python,python-sync,cherami
```

Comment:
@breerly I'm wondering how big deal it is to have the 40 seconds startup for cassandra/cherami.

I tried to start cassandra and cherami in seperate dockers hoping to bring down the total startup time, but it's more complicated than I thought(mostly around pointing cherami to use the correct cassandra IP). While it's definitely doable but it'll require some code changes on cherami side.

On the other hand, The minimum time required to start both is 20+ seconds(so it won't be 1 second). I'm wondering whether you guys are running 'make crossdock' for every commit or just 'make test' because we can skip the cherami test for 'make test' and only have it for 'make crossdock' so it's only running in CI.